build/_build/Covertura/CompareReports.cs

```diff
@@ -0,0 +1,344 @@
+                                           .ToDictionary(x => $"{x.Filename}_{x.Name}", x => x),
+                              })
+                             .ToDictionary(x => x.Name, x => x);
+
+            return report;
+        }
+
+        public static CoverturaReportComparison Compare(CoverturaReport oldReport, CoverturaReport newReport)
+        {
+            var comparison = new CoverturaReportComparison
+            {
+                Old = oldReport,
+                New = newReport,
+                LineCoverageChange = decimal.Round(newReport.LineRate - oldReport.LineRate, decimals: 2),
+                BranchCoverageChange = decimal.Round(newReport.BranchRate - oldReport.BranchRate, decimals: 2),
+                ComplexityChange = newReport.Complexity - oldReport.Complexity,
+            };
+
+            var matchedPackages = new Dictionary<string, CoverturaReportComparison.PackageChanges>();
+
+            foreach (var kvp in oldReport.Packages)
+            {
+                var oldPackage = kvp.Value;
+                // try and find package in other report
+                if (!newReport.Packages.TryGetValue(kvp.Key, out var newPackage))
+                {
+                    // package was removed
+                    comparison.RemovedPackages.Add(oldPackage);
+                    continue;
+                }
+
+                // do class-level comparison
+                var changes = new CoverturaReportComparison.PackageChanges
+                {
+                    Old = oldPackage,
+                    New = newPackage,
+                    LineCoverageChange = decimal.Round(newPackage.LineRate - oldPackage.LineRate, decimals: 2),
+                    BranchCoverageChange = decimal.Round(newPackage.BranchRate - oldPackage.BranchRate, decimals: 2),
+                    ComplexityChange = newPackage.Complexity - oldPackage.Complexity,
+                };
+
+                foreach (var classKvp in oldPackage.Classes)
+                {
+                    var oldClass = classKvp.Value;
+                    if (!newPackage.Classes.TryGetValue(classKvp.Key, out var newClass))
+                    {
+                        changes.RemovedClasses.Add(oldClass);
+                        continue;
+                    }
+
+                    var changeSummary = new CoverturaReportComparison.ClassChanges
+                    {
+                        Name = oldClass.Name,
+                        Filename = oldClass.Filename,
+                        LineCoverageChange = decimal.Round(newClass.LineRate - oldClass.LineRate, decimals: 2),
+                        BranchCoverageChange = decimal.Round(newClass.BranchRate - oldClass.BranchRate, decimals: 2),
+                        ComplexityChange = newClass.Complexity - oldClass.Complexity,
+                    };
+                    // somewhat arbitrary
+                    changeSummary.IsSignificantChange = Math.Abs(changeSummary.LineCoverageChange) > 0.05m
```

Comment:
I'm thinking the `0.05m` value can be abstracted into a constant so we can easily keep track of our decision criteria

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1704/commits/8a7db0e5b58679872b803b47c25434462ba93783